docs/cloud/dbaas/postgres-extensions.md

```diff
@@ -13,61 +13,63 @@
 ## Currently available extensions
 
 If there are extensions missing here that you would like to see available in Pukki, please contact the [CSC Service Desk](../../support/contact.md).
+Note that the extension and versions might differ between postgresql versions.
```

Comment:
Maybe `Note that the available extensions and their versions might differ between PostgreSQL versions.`

Could also indicate which PSQL version's extensions list this is.

Reply:
I removed the extension versions so that I could just refer to the "latest PostgreSQL version" so that the documentation would not get outdated too quickly.